spaces/S000045/properties/P000216.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S000045
+property: P000216
+value: false
+---
+
+The open cover $\{(-t,t): t\in(0,1) \}$ of the subspace $(-1,1)$ has no finite open refinement and since $0$ is a generic point, the refinement cannot be locally finite at $0$.
```

Comment:
```suggestion
The open cover $\{(-t,t): t\in(0,1) \}$ of the subspace $(-1,1)$ has no finite open refinement and since $0$ is a generic point, any infinite refinement cannot be locally finite at $0$.
```